src/anemoi/graphs/plotting/interactive_html.py

```diff
@@ -0,0 +1,234 @@
+    """
+    source_name, _, target_name = edges_to_plot
+    edge_x, edge_y = edge_list(graph, source_nodes_name=source_name, target_nodes_name=target_name)
+    assert source_name in graph.node_types, f"edges_to_plot ({source_name}) should be in the graph"
+    assert target_name in graph.node_types, f"edges_to_plot ({target_name}) should be in the graph"
+    lats_source_nodes, lons_source_nodes = node_list(graph, source_name)
+    lats_target_nodes, lons_target_nodes = node_list(graph, target_name)
+    dst_num_nodes = graph[target_name].num_nodes
+    node_adjacencies, node_text = compute_node_adjacencies(graph, source_name, target_name, dst_num_nodes)
+
+    edge_trace = go.Scattergeo(
+        lat=edge_x,
+        lon=edge_y,
+        line={"width": 0.5, "color": "#888"},
+        hoverinfo="none",
+        mode="lines",
+        name="Connections",
+    )
+
+    source_node_trace = go.Scattergeo(
+        lat=lats_source_nodes,
+        lon=lons_source_nodes,
+        mode="markers",
+        hoverinfo="text",
+        name=source_name,
+        marker={
+            "showscale": False,
+            "color": "red",
+            "size": 2,
+            "line_width": 2,
+        },
+    )
+
+    target_node_trace = go.Scattergeo(
+        lat=lats_target_nodes,
+        lon=lons_target_nodes,
+        mode="markers",
+        hoverinfo="text",
+        name=target_name,
+        text=node_text,
+        marker={
+            "showscale": True,
+            "colorscale": "YlGnBu",
+            "reversescale": True,
+            "color": node_adjacencies,
+            "size": 10,
+            "colorbar": {"thickness": 15, "title": "Node Connections", "xanchor": "left", "titleside": "right"},
+            "line_width": 2,
+        },
+    )
+    layout = go.Layout(
+        title="<br>" + f"Graph {source_name} --> {target_name}",
+        titlefont_size=16,
+        showlegend=True,
+        hovermode="closest",
+        margin={"b": 20, "l": 5, "r": 5, "t": 40},
+        annotations=[annotations_style],
+        legend={"x": 0, "y": 1},
+        xaxis=plotly_axis_config,
+        yaxis=plotly_axis_config,
+    )
+    fig = go.Figure(data=[edge_trace, source_node_trace, target_node_trace], layout=layout)
+    fig.update_geos(fitbounds="locations")
+
+    if out_file is not None:
+        fig.write_html(out_file)
+    else:
+        fig.show()
+
+
+def plot_isolated_nodes(graph: HeteroData, out_file: Optional[Union[str, Path]] = None) -> None:
+    """Plot isolated nodes.
+
+    This method creates an interactive visualization of the isolated nodes in the graph.
+
+    Parameters
+    ----------
+    graph : AnemoiGraph
+        The graph to plot.
+    out_file : str | Path, optional
+        Name of the file to save the plot. Default is None.
+    """
+    isolated_nodes = compute_isolated_nodes(graph)
+
+    if len(isolated_nodes) == 0:
+        LOGGER.warning("No orphan nodes found.")
```

Comment:
To be consistent, I would replace this with 'isolated nodes' rather than orphans